internal/installer/reinstall.go

```diff
@@ -49,6 +53,20 @@ func (i *installer) WriteReinstallData(ctx context.Context, tmpWorkDir, isoWorkD
 	return nil
 }
 
+func validateReinstall(newConfig, secretConfig *imagebased.SeedReconfiguration) error {
+	if newConfig.BaseDomain != secretConfig.BaseDomain {
+		return fmt.Errorf("provided base domain (%s) must match previous base domain (%s)", newConfig.BaseDomain, secretConfig.BaseDomain)
+	}
+	if newConfig.ClusterName != secretConfig.ClusterName {
+		return fmt.Errorf("provided cluster name (%s) must match previous cluster name (%s)", newConfig.ClusterName, secretConfig.ClusterName)
+	}
+	if newConfig.Hostname != secretConfig.Hostname {
```

Comment:
i wonder if hostname is important actually, it doesn't influence any key so why not to allow changing it?

Reply:
Wouldn't we need the same hostname for the existing kubeconfig to still work?

Reply:
Oh I guess that's a different dns entry. :shrug: